src/normal.c

```diff
@@ -6544,11 +6548,32 @@ nv_dot(cmdarg_T *cap)
 }
 
 /*
- * CTRL-R: undo undo
+ * CTRL-R: undo undo or specify register in select mode
  */
     static void
 nv_redo(cmdarg_T *cap)
```

Comment:
Does the name of this function need change?

Reply:
Fixed.